plugins/backend-trace-forwarder/src/Cardano/BM/Backend/TraceForwarder.lhs

```diff
@@ -300,24 +300,6 @@ sendItems config tfMVar items@(lo:_) getStateDigest =
     let bs' = BL.toStrict $ encode a
     in (BS.length bs', bs')
 
-  -- Handle is bad, it looks like the connection is broken.
-  -- Check if the queue is almost full.
-  reConnectIfQueueIsAlmostFull = do
-    currentTF <- readMVar tfMVar
-    currentQueueSize <- atomically $ TBQ.lengthTBQueue (tfQueue currentTF)
-    when (queueIsAlmostFull currentQueueSize) $ do
-      -- The queue is almost full, it means that log items will be dropped soon.
-      -- Initiate re-establishing of connection.
-      closeHandle $ tfHandle currentTF
```

Comment:
This line was a problem: if the `handle` is bad (because of interrupted connection with `TraceAcceptorBK`) we shouldn't close it! Otherwise, the thread will be **blocked** on it.

Reply:
But we can't be leaking handles/file descriptors either.

We need to think of some approach for proper cleanup.

Reply:
As an off-hand solution -- we could spawn a thread that would do `closeHandle`.